etcd/fs/fs_test.go

```diff
@@ -183,123 +183,81 @@ func TestFs1(t *testing.T) {
 	}
 }
 
-func TestFs2(t *testing.T) {
```

Comment:
Why are you nuking these??

Reply:
The two tests are identical, and we needed more cases, so I combined them into a single test, and added two more test cases. TestEtcdCopyFs, does the same tests as TestFs2 and TestFs3, but adds one case for a recursive copy and one extra deep copy.

Reply:
I've been to lax on this, and judging by my recent commit, I'm obviously not being a good example, so this is a good time to start: if you'd like to make all of those changes, that's fine, but they need to be each done in separate commits (and reviewed individually).

Reply:
@purpleidea: I split up the commit, so I add the new test in one, and replace TestFs2 and TestFs3 in the second one.